cmd/controller/app/server.go

```diff
@@ -129,53 +86,9 @@ func Run(s *ServerRunOptions) error {
 		return err
 	}
 
-	run := func(ctx context.Context) {
-		setupLog.Info("starting manager")
-		if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-			setupLog.Error(err, "unable to start manager")
-			panic(err)
-		}
-
-		select {}
-	}
-	schedInformerFactory.Start(stopCh)
-	coreInformerFactory.Start(stopCh)
-	if !s.EnableLeaderElection {
```

Comment:
this seems to be totally disabling leader election?

Reply:
No need to implement this manually, the ctrl runtime can handle this instead. We could enable leader election when create `ctrl.NewManager`

Reply:
https://github.com/kubernetes-sigs/scheduler-plugins/blob/aeb7ff27f5fa6559c588d7a133fe933d1fc65ca6/cmd/controller/app/server.go#L93-L100